package.json

```diff
@@ -4,14 +4,19 @@
   "main": "dist/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "ncc build src/index.ts --license LICENSES.txt"
+    "build": "ncc build src/index.ts --license LICENSES.txt",
+    "format": "prettier --write '**/*.ts'",
+    "format-check": "prettier --check '**/*.ts'",
+    "prepare": "husky install"
   },
   "dependencies": {
     "@actions/core": "1.2.6",
     "@actions/github": "4.0.0"
   },
   "devDependencies": {
     "@vercel/ncc": "0.27.0",
-    "typescript": "4.1.5"
+    "prettier": "2.2.1",
+    "typescript": "4.1.5",
+    "husky": "^6.0.0"
```

Comment:
This should not have the `^`

Reply:
sure, that's actually a preference too. I like semver bands in my repo though I understand I am then in non-reproducible builds territory. For really solid reasons lots of people don't like that. I'll remove it

Reply:
removed, rebased to current state of main branch, re pushed, including with content change based on trailing comma setting difference